resources/js/visual-editor/blocks/callout/callout.css

```
@@ -0,0 +1,59 @@
+
+.ap-callout__icon {
+    flex: 0 0 auto;
+    width: 1.5rem;
+    height: 1.5rem;
+    color: var(--ap-callout-accent, currentColor);
+}
+
+.ap-callout__icon svg {
+    width: 100%;
+    height: 100%;
+    fill: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)